src/main/c/Objects/pyjmethod.c

```diff
@@ -244,19 +244,41 @@ int PyJMethod_CheckArguments(PyJMethodObject* method, JNIEnv *env,
     return matchTotal;
 }
 
-// pyjmethod_call. where the magic happens.
-//
-// okay, some of the magic -- we already the methodId, so we don't have
-// to reflect. we just have to parse the arguments from python,
-// check them against the java args, and call the java function.
-//
-// easy. :-)
+
+/*
+ * This function contains much of the same logic as pyjconstructor_call. Ensure
+ * any changes here are done there if needed. If you reading this and see a way
+ * to reduce the redundancy please do.
+ *
+ * This function primarily does two things.
+ *  1) Convert the python method arguments to java objects
+ *  2) Call a Java method
+ *
+ * Several different factors make the conversion of args complicated:
+ *  1) Python includes a reference to this/self as the first arg but Java needs
+ *     this/self separate from the other args.
+ *  2) Varargs are appended to the args by Python but Java varargs are a
+ *     seperate array.
```

Comment:
seperate -> separate.  You spelled it right just above.

Reply:
Fixed